core/base/abstractTriangulation/AbstractTriangulation.h

```diff
@@ -2768,10 +2773,24 @@ namespace ttk {
       return &(this->neighborRanks_);
     }
 
+    virtual inline const std::vector<std::vector<SimplexId>> *
+      getGhostCellsPerOwner() const {
+      return &(this->ghostCellPerOwner_);
+    }
+
+    virtual inline const std::vector<std::vector<SimplexId>> *
+      getRemoteGhostCells() const {
+      return &(this->remoteGhostCells_);
+    }
+
     virtual inline void setHasPreconditionedDistributedVertices(bool flag) {
       this->hasPreconditionedDistributedVertices_ = flag;
     }
 
+    virtual inline bool getHasPreconditionedDistributedCells() const {
```

Comment:
can you please simply rename this function into `hasPreconditionedDistributedCells()` (to follow the rest of the preconditioning API)? thanks!